scania-digital-design-system/tegel
# fix: align input text colors

## **Describe pull-request**  
This PR aligns the text colors of labels, placeholders and input text for Text Field, Textarea and Dropdown components. Additionally updated the label text for Toggle in dark mode. These alignments are not present in the UI Library yet but have been verified with a designer. For testing reference, the following is considered true:

- Input text in light mode should be grey-958
- Input text in dark mode should be grey-50
- Labels (both inside and outside position) in light mode should be grey-958
- Labels (both inside and outside position) in dark mode should be grey-50
- Placeholders in light mode should be grey-500
- Placeholders in dark mode should be grey-600

## **Issue Linking:**  
- **Jira:** [CDEP-3563](https://tegel.atlassian.net/browse/CDEP-3563)   

## **How to test**  
1. Go to Storybook link
2. Check in Text Field, Textarea, Dropdown and Toggle
3. Verify that the above parts of components have the correct values

## **Checklist before submission**

## **Suggested test steps**

## **Additional context**  
This PR is a re-work of the content in [this PR](https://github.com/scania-digital-design-system/tegel/pull/823) since it got muddled with other files that shouldn't be there and was therefore closed.


[CDEP-3563]: https://tegel.atlassian.net/browse/CDEP-3563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
